eks-cluster.tf

```diff
@@ -56,6 +56,24 @@ module "eks" {
       capacity_type  = local.config.capacity_type
       k8s_labels     = local.standard_tags
 
+      update_config = {
+        max_unavailable_percentage = local.config.max_unavailable_percentage # or set `max_unavailable`
+      }
+    }
+    apps-public = {
```

Comment:
I definitely don't like the `public` name, it's way too general and does not make any sense in this context.

It is expected to run only nodes in this group right? While `nodes` would be tautology (because of k8s nodes), I'd suggest `aenodes`, `blockchain` or something in that direction.